boundlessgeo_schema/README.rst

```diff
@@ -0,0 +1,22 @@
+boundlessgeo-schema
+-------------------
+
+This package will convert the packaged actions.json to a dictionary.
+By convention, Actions can be Queries or Commands in a
+[CQRS](https://martinfowler.com/bliki/CQRS.html) architecture.
```

Comment:
Should this be `CQRS
<https://martinfowler.com/bliki/CQRS.html>`_.
b/c it's .rst and not .md?